Objective-C/Tests/MessageEndpointTest.m

```diff
@@ -200,7 +200,8 @@ - (void) startDiscovery {
     _browser.delegate = self;
     [_browser startBrowsingForPeers];
     
-    [self waitForExpectations: @[_clientConnected, _serverConnected] timeout: 10.0];
+    // cool down period(disconnected to next connected state), is taking around 4-10secs
+    [self waitForExpectations: @[_clientConnected, _serverConnected] timeout: 30.0];
```

Comment:
Do you know why it is talking a lot of time to connect? Is there a delay or sleep in the code?